packages/config/config/devices/0x024e/WIN-GDC-02.json

```diff
@@ -0,0 +1,323 @@
+{
+	"manufacturer": "zConnect",
+	"manufacturerId": "0x024e",
+	"label": "DHS-WIN-GDC-02",
+	"description": "Garage Door Opener",
+	"devices": [
+		{
+			"productType": "0x0021",
+			"productId": "0x0014"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"endpoints": {
+		"0": {
+			"associations": {
+				"1": {
+					"label": "Lifeline",
+					"maxNodes": 3,
+					"isLifeline": true
+				},
+				"2": {
+					"label": "Sensor 1",
+					"maxNodes": 10
+				},
+				"3": {
+					"label": "Sensor 2",
+					"maxNodes": 10
+				},
+				"4": {
+					"label": "Sensor 3",
+					"maxNodes": 10
+				},
+				"5": {
+					"label": "Sensor 4 (optional temp sensor)",
+					"maxNodes": 10
+				}
+			}
+		},
+		"1": {
+			"associations": {
+				"1": {
+					"$import": "#endpoints/0/associations/1",
+					"isLifeline": false
+				},
+				"2": {
+					"$import": "#endpoints/0/associations/2"
+				}
+			}
+		},
+		"2": {
+			"associations": {
+				"1": {
+					"$import": "#endpoints/0/associations/1",
+					"isLifeline": false
+				},
+				"2": {
+					"$import": "#endpoints/0/associations/3"
+				}
+			}
+		}
+	},
+	"paramInformation": {
+		"1": {
+			"$import": "templates/zconnect_template.json#relay_mode_selection",
+			"label": "Relay 1 Mode"
+		},
+		"2": {
+			"$import": "templates/zconnect_template.json#interval_1-65000",
+			"label": "Strobe Closed Interval",
+			"description": "Relay 1 and 2 closed contacts time interval in strobe mode.",
+			"defaultValue": 10
+		},
+		"3": {
+			"$import": "templates/zconnect_template.json#interval_1-65000",
+			"label": "Strobe Open Interval",
+			"description": "Relay 1 and 2 open contacts time interval in strobe mode.",
+			"defaultValue": 10
+		},
+		"4": {
+			"$import": "templates/zconnect_template.json#interval_1-65000_disable",
+			"label": "Relay 1 Auto Off Delay",
+			"description": "Relay will turn off automatically after time specified in this parameter. After receiving command On Relay 1 will be switched Off automatically with predefined delay."
+		},
+		"5": {
+			"$import": "templates/zconnect_template.json#interval_1-65000_disable",
+			"label": "Relay 1 On Delay",
+			"description": "Relay 1 will be switched On with predefined delay."
+		},
+		"6": {
+			"$import": "templates/zconnect_template.json#interval_1-65000_disable",
+			"label": "Relay 1 Off Delay",
+			"description": "Relay 1 will be switched Off with predefined delay."
+		},
+		"7": {
+			"$import": "../templates/master_template.json#state_after_power_failure_prev_off",
+			"label": "Relay 1 Restore After Power Loss",
+			"description": "Restore to the last known state for relay 1 after power was interrupted."
+		},
+		"8": {
+			"label": "Broadband Command Action",
```

Comment:
I'm unclear what broadband mean. Broadcast? @AlCalzone

Reply:
Probably, not sure. Copied this from the manual. Broadcast seems sensible.